typings/nerdamer-defs.d.ts

```diff
@@ -0,0 +1,48 @@
+// Code - https://github.com/jiggzson/nerdamer/blob/master/nerdamer.core.js
+
+interface Expression {
+    // Returns all variables in the given expression.
+    variables: () => String[];
+
+    // Returns the text representation of the expression.
+    text: (option: String, n: String) => strings;
```

Comment:
Is `strings` a valid type? Also can you please use `string` instead of `String` and `boolean` instead of `Boolean`.

Reply:
Also you've made a change in package.json. So there should be a change in `yarn.lock` too, right?

Reply:
Just curious - why are we restricting String to string and ditto for boolean. And if we are doing that, we should have an automated test for that. WDYT?

Reply:
`String` is actually a js object used to create strings like
```typescript
const a = String('hello');
```

but `string` is the typescript string type.

Reference - https://stackoverflow.com/questions/14727044/typescript-difference-between-string-and-string

Yup, I agree there should be a check for that!

Reply:
Done, thanks!